core/capabilities/syncer.go

```diff
@@ -37,65 +53,99 @@ func NewRegistrySyncer(peerWrapper p2ptypes.PeerWrapper, registry types.Capabili
 
 func (s *registrySyncer) Start(ctx context.Context) error {
 	// NOTE: temporary hard-coded DONs
-	defaultStreamConfig := p2ptypes.StreamConfig{
-		IncomingMessageBufferSize: 1000000,
-		OutgoingMessageBufferSize: 1000000,
-		MaxMessageLenBytes:        100000,
-		MessageRateLimiter: ragep2p.TokenBucketParams{
-			Rate:     10.0,
-			Capacity: 1000,
-		},
-		BytesRateLimiter: ragep2p.TokenBucketParams{
-			Rate:     10.0,
-			Capacity: 1000,
-		},
-	}
-	peerIDs := []string{
+	workflowDONPeers := []string{
 		"12D3KooWF3dVeJ6YoT5HFnYhmwQWWMoEwVFzJQ5kKCMX3ZityxMC",
 		"12D3KooWQsmok6aD8PZqt3RnJhQRrNzKHLficq7zYFRp7kZ1hHP8",
 		"12D3KooWJbZLiMuGeKw78s3LM5TNgBTJHcF39DraxLu14bucG9RN",
 		"12D3KooWGqfSPhHKmQycfhRjgUDE2vg9YWZN27Eue8idb2ZUk6EH",
 	}
-	peers := make(map[ragetypes.PeerID]p2ptypes.StreamConfig)
-	donInfo := &remotetypes.DON{
-		ID: "don1",
+	capabilityDONPeers := []string{
+		"12D3KooWHCcyTPmYFB1ydNvNcXw5WyAomRzGSFu1B7hpB4yi8Smf",
+		"12D3KooWPv6eqJvYz7TcQWk4Y4XjZ1uQ7mUKahdDXj65ht95zH6a",
 	}
-	for _, peerID := range peerIDs {
-		var p ragetypes.PeerID
-		err := p.UnmarshalText([]byte(peerID))
-		if err != nil {
-			return err
+	allPeers := make(map[ragetypes.PeerID]p2ptypes.StreamConfig)
+	process := func(peers []string, donInfo *remotetypes.DON) error {
```

Comment:
@bolekk Might be better to extract this into a helper function with a descriptive name :)

Reply:
This is only temporary for hardcoded Syncer values so I wouldn't worry about it too much. Let me add a more descriptive name here.